platform/asterix/boot/src/drivers/pmic.c

```diff
@@ -38,6 +62,24 @@ void pmic_init(void) {
 
   nrfx_twi_enable(&twi);
 
+  // Configure charger (TODO: values are board/battery dependent)
```

Comment:
OK to defer the details of this until later but please file a Linear or GitHub issue to update this particular logic, and cite it in the comment.  4v2 is safe in the mean time and 64 mA should be like 0.5C on the plan of record battery, which will take forever but is also safe

Reply:
Bootloader is already super hacky, I have an issue to implement a single, portable bootloader on the roadmap.

Reply:
Yeah, though in this particular case, the reminder issue is "remember to tune the code specifically in the bootloader for our specific battery", and needs to block HW ship.

Related to FIRM-84, I guess, but not quite the same.